src/Symfony/Component/VarDumper/Caster/MemcachedCaster.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class MemcachedCaster
+{
+    public static function castMemcached(\Memcached $c, array $a, Stub $stub, $isNested)
+    {
+        $a += array(
+            Caster::PREFIX_VIRTUAL.'servers' => $c->getServerList(),
+            Caster::PREFIX_VIRTUAL.'options' => new EnumStub(
+                array(
+                    'OPT_COMPRESSION' => $c->getOption(\Memcached::OPT_COMPRESSION),
```

Comment:
just an idea to reduce this array: we could list only values that have a different value than the default one? wdyt?

Reply:
I like this idea. But we should maybe rename this property to `non_default_options`?